src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/** 
+ * This class represents the coordinates of a site location provided by NSW GOV API for each site avaialble for fitering.
```

Comment:
Is there any reason this class can't be used for coordinates in general (i.e. they don't have to be related to NSW GOV functionality)?

Reply:
oh nope! this can be the coordinates in general. I mainly made it for it to be a class for GetCoord function, but this class can be used for coordinates in general